docs/core/extensions/dependency-injection-usage.md

```diff
@@ -98,10 +98,10 @@ Update *Program.cs* with the following code:
 
 :::code source="snippets/configuration/console-di/Program.cs" id="Program" highlight="8-11":::
 
-Each `services.Add{LIFETIME}<{SERVICE}>` extension method adds (and potentially configures) services. We recommend that apps follow this convention. Place extension methods in the <xref:Microsoft.Extensions.DependencyInjection?displayProperty=fullName> namespace to encapsulate groups of service registrations. Including the namespace portion `Microsoft.Extensions.DependencyInjection` for DI extension methods also:
+Each `services.Add{LIFETIME}<{SERVICE}>` extension method adds (and potentially configures) services. We recommend that apps follow this convention. Don't place extension methods in the <xref:Microsoft.Extensions.DependencyInjection?displayProperty=fullName> namespace unless you're authoring an official Microsoft package. When extension methods are defined within the `Microsoft.Extensions.DependencyInjection` namespace it:
 
-- Allows them to be displayed in [IntelliSense](/visualstudio/ide/using-intellisense) without adding additional `using` blocks.
-- Prevents excessive `using` statements in the `Program` or `Startup` classes where these extension methods are typically called.
+- allows them to be displayed in [IntelliSense](/visualstudio/ide/using-intellisense) without adding additional `using` blocks
```

Comment:
It was correct that these were capitalized and full-stopped before - see https://styleguides.azurewebsites.net/Styleguide/Read?id=2696&topicid=28112.